README.md

```diff
@@ -30,9 +30,9 @@ Glueby has below features.
 
 ```ruby
 
-Glurby.configuration do |c|
-  c.wallet_adapter = :core
-  c.rpc_config = { schema: 'http', host: '127.0.0.1', port: 12381, user: 'user', password: 'pass' }
+Glurby.configure do |config|
+  config.wallet_adapter = :activerecord
+  config.rpc_config = { schema: 'http', host: '127.0.0.1', port: 12381, user: 'user', password: 'pass' }
```

Comment:
When using `:activerecord`,  `rpc_config` does not necessary.

Reply:
It is necessary because broadcasting uses `sendrawtransaction` RPC always.